package.json

```diff
@@ -14,7 +14,7 @@
     "eslint",
     "config"
   ],
-  "author": "Joe Lencioni",
+  "author": "Brigade Group, Inc.",
```

Comment:
"Brigade Group, Inc." from the [ToS](https://www.brigade.com/tos) which I would hope is accurate ❓ 😟